EyeOnBuzz/Screens/Support/BackButton.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  BackButton.swift
+//  EyeOnBuzz
+//
+//  Created by Weverton Couto Timoteo on 1/29/18.
+//  Copyright © 2018 Weverton Couto Timoteo. All rights reserved.
+//
+
+import UIKit
+
+class BackButton: UIBarButtonItem {
+    
+    class func build(title: String = "") -> BackButton {
+        let backButton: BackButton = BackButton(title: title, style: UIBarButtonItemStyle.plain, target: nil, action: nil)
+        
+        backButton.tintColor = Colors.byProperty("backButton")
```

Comment:
why did you choose to use Strings here 👆? Are there any downsides to that?

Reply:
Did you mean to use Strings in `byProperty` method argument? The problem is to try to get a nonexistent property in `Dictionary`, with more time I would rewrite to use enums, fixing this problem (and avoiding creating a "new" string for each call).